File: JAVAday2/GroceryManagerApp.java
import java.util.Arrays;
import java.util.Scanner;
public class GroceryManagerApp{
public static void main(String...args){

        
Scanner input = new Scanner(System.in);

while (true) {
System.out.println("1. Add Product");
System.out.println("2. Remove Item");
System.out.println("3. Show All Items");
System.out.println("0. Exit");
System.out.println("Enter your menu: ");
    

int item = input.nextInt();
String menu = input.nextLine();
     
switch (item) {
    case 1:

        System.out.print("Enter item to add: ");
        String addProduct = input.nextLine();

        productItem.add(addProduct);
        System.out.println(addProduct + " added"); 
    break;

    case 2:
        System.out.print("Enter item to remove: ");
        String removeProduct = input.nextLine();
        if (productItem.remove(removeProduct)) {
        System.out.println(removeProduct + " removed.");
        } else {
        System.out.println("Item not found.");
          }
          break;







    case 0:
        System.out.println("Exiting program...");
            break;









            }
        }
    }
}




















































    

File: PYTHONday/Grocery_Manager_App.py

product_items = []

while True:
    print("1. Add Product")
    print("2. Remove Item")
    print("3. Display All Items")
    print("0. Exit")
    menu = input("Enter your menu: ")

    if menu == "1":
            add_product = input("Enter item to add: ")
            product_items.append(add_product)
            print(f"{add_product} added")

    elif menu == "2":
            remove_product = input("Enter item to remove: ")
            if remove_product in product_items:
                product_items.remove(remove_product)
                print(f"{remove_product} removed.")
            else:
                print("Item not found.")

    elif menu == "3":
            print("Display All Items:")
            if product_items:
                for item in product_items:
                    print( item)
            else:
                print("No items.")

    elif menu == "0":
            print("Exit")
            break

        
        

